modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -117,3 +117,36 @@ function perflab_aao_autoloaded_options_size() {
 	global $wpdb;
 	return (int) $wpdb->get_var( 'SELECT SUM(LENGTH(option_value)) FROM ' . $wpdb->prefix . 'options WHERE autoload = \'yes\'' );
 }
+
+/**
+ * Fetch autoload top list.
+ *
+ * @since n.e.x.t
+ *
+ * @return object autoloaded data - autoload size, entries, option and its names.
+ */
+function perflab_aao_autoloaded_options() {
+	global $wpdb;
+	$autoload_summary                   = $wpdb->get_row( "SELECT SUM(LENGTH(option_value)) AS autoload_size,COUNT(*) AS autload_entries FROM `$wpdb->options` WHERE autoload='yes'" );
+	$autoload_summary->autoload_toplist = $wpdb->get_results( "SELECT option_name, LENGTH(option_value) AS option_value_length FROM `wp_options` WHERE autoload='yes' AND LENGTH(option_value) > 100 ORDER BY option_value_length DESC LIMIT 20" );
+	return $autoload_summary;
+}
+
+/**
+ * Get formatted autoload options table.
+ *
+ * @since n.e.x.t
+ *
+ * @return string HTML formatted table.
+ */
+function perflab_aao_autoloaded_options_table() {
+	$autoload_summary = perflab_aao_autoloaded_options();
+
+	$html_table = '<table class="widefat striped"><thead><tr><th scope="col">#</th><th scope="col">Option Name</th><th scope="col">Size</th></tr></thead><tbody>';
+	foreach ( $autoload_summary->autoload_toplist as $key => $value ) {
+		$html_table .= sprintf( '<tr><td> %s </td><td> %s </td><td> %s </td><tr>', $key + 1, $value->option_name, $value->option_value_length );
```

Comment:
Do we need the spaces around all the values? How about
```suggestion
		$html_table .= sprintf( '<tr><td>%s</td><td>%s</td><td>%s</td><tr>', $key + 1, $value->option_name, $value->option_value_length );
```